examples/formats/json/python/func.py

```diff
@@ -0,0 +1,34 @@
+# All Rights Reserved.
```

Comment:
Remove all of these.

Reply:
They are required by apache 2.0 licence, no?